pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.black]
+line_length = 79
```

Comment:
You can use 120

Reply:
Ok, I decided to try and made changes to my config files.